secret-store/test/test_secret_store.py

```diff
@@ -0,0 +1,67 @@
+import os
+import pytest
+from pathlib import Path
+from secret_store import Secrets, Credentials
+from sqlcipher3 import dbapi2 as sqlcipher
+
+
+@pytest.fixture
+def sample_file(tmp_path) -> Path:
```

Comment:
```suggestion
def sample_file(tmp_path: Path) -> Path:
```
also for other functions